packages/did-manager/package.json

```diff
@@ -9,7 +9,8 @@
     "extract-api": "yarn veramo dev extract-api"
   },
   "dependencies": {
-    "@veramo/core": "^1.2.0"
+    "@veramo/core": "^1.2.0",
+    "@veramo/did-discovery": "^1.2.0"
```

Comment:
I think the basic discovery providers should be implemented in the `did-discovery` package.
This would reverse the dependency relationship between plugins, or even removing the need for new coupling.

Reply:
I'm not sure about this. If we reverse dependency relationships between plugins, you will end up in a situation where you want to use `DIDDiscovery` plugin with only a specific provider in an environment with no DB, but you will be forced to install `TypeORM`, because it would be a dependency through `@veramo/data-store`. At the moment `DIDDiscovery` plugin is very light weight.